CriteoPublisherSdk/Sources/Network/Serializers/CR_BidRequestSerializer.m

```diff
@@ -48,8 +45,8 @@ - (instancetype)initWithGdprSerializer:(CR_GdprSerializer *)gdprSerializer {
 
 #pragma mark - Public
 
-- (NSURL *)urlWithConfig:(CR_Config *)config {
-  NSString *query = [NSString stringWithFormat:@"profileId=%@", config.profileId];
+- (NSURL *)urlWithProfileId:(NSNumber *)profileId config:(CR_Config *)config {
```

Comment:
Profile ID is useless in the query string. What do you think about taking the opportunity to remove it ? It may seems FAR but I think that injecting the new profile id there add too many useless stuff.

See the [schema on CDB](http://review.crto.in/gitweb?p=publisher/direct-bidder.git;a=blob;f=directbidder-app/src/main/scala/com/criteo/directbidder/inapp/InApp.scala;h=4e38f9e8da2e0d5ce8c78d8430313466acb0710c;hb=HEAD#l37)

Reply:
Done.